src/commands/force/user/list.ts

```diff
@@ -47,7 +47,7 @@ export class UserListCommand extends SfdxCommand {
       // if they passed in a alias and it maps to something we have an Alias.
       const alias = aliases.getKeysByValue(authData.getUsername())[0];
       const configAgg = ConfigAggregator.getInstance();
-      const defaultUserOrAlias = configAgg.getLocalConfig().get('defaultusername');
+      const defaultUserOrAlias = configAgg.getLocalConfig()?.get('defaultusername');
       return {
         defaultMarker: defaultUserOrAlias === username || defaultUserOrAlias === alias ? '(A)' : '',
```

Comment:
Should we show a marker for both the default username or the default devhub?

Reply:
Unlike `org:list`, for `user:list` it means the main (admin) user on the org, as opposed to other users that were created by the main user via `user:create`

Reply:
welp, that's what it's supposed to do...but it looks like we're looking at what you're describing.  So I'm gonna fix that, too.

>DESCRIPTION
  The original scratch org admin is marked with "(A)"